denops_std/helper/load_test.ts

```diff
@@ -17,10 +17,54 @@ test({
     await load(
       denops,
       new URL(
-        "https://raw.githubusercontent.com/vim-denops/deno-denops-std/main/denops_std/helper/load_test.vim",
+        "https://raw.githubusercontent.com/vim-denops/deno-denops-std/4df988237b3ea04aaa665974b44c8e7df8e237cb/denops_std/helper/load_test.vim",
         import.meta.url,
       ),
     );
     assertEquals(await denops.eval("g:denops_std_load_test") as number, 1);
   },
 });
+
+test({
+  mode: "any",
+  name: "load() load not exists local Vim script file",
+  fn: async (denops) => {
+    const url = new URL("./load_test_not_exists.vim", import.meta.url);
+    let filePath: string;
+    if (Deno.build.os === "windows") {
+      filePath = url.pathname.slice(1).replaceAll("/", "\\\\");
+    } else {
+      filePath = url.pathname;
+    }
+    await assertThrowsAsync(
+      async () => {
+        await load(
+          denops,
+          new URL("./load_test_not_exists.vim", import.meta.url),
+        );
+      },
+      undefined,
+      `Failed to call 'call' with ["denops#api#cmd","call execute(l:__denops_internal_command, '')",{"scriptPath":"${filePath}","__denops_internal_command":["execute printf('source %s', fnameescape(scriptPath))"]}]: Error: Failed to call 'nvim_call_function' with ["denops#api#cmd",["call execute(l:__denops_internal_command, '')",{"scriptPath":"${filePath}","__denops_internal_command":["execute printf('source %s', fnameescape(scriptPath))"]}]]: [0,"Vim(source):E484: Can't open file ${filePath}`,
```

Comment:
You don't need to specify full sentence here.

```
        `Failed to call 'call' with`,
```

seems enough.

Reply:
Fixed it with c1fc317.